bin/pint

```diff
@@ -150,15 +150,28 @@ if command_args['--region']:
 
 try:
     if command_args['images']:
-        data = ifsrequest.get_image_data(
-            framework,
-            image_state,
-            output_format,
-            region,
-            command_args['--filter'])
-        if '<images/>' in data or '"images": []' in data:
-            data = 'No information available. Please check your filter'
-        print(data)
+        no_info_regions = []
+        for reg in region.split(','):
```

Comment:
This seems to make region a required arg.

Reply:
region is `all` by default
I do not understand what you mean

Reply:
Ack I did not see the default being set